exercises/robot-simulator/test/Tests.hs

```diff
@@ -44,21 +44,21 @@ specs = do
             it "shouldn't change position" $
               coordinates (move robot inst) `shouldBe` (0, 0)
 
-    describe "turning right rotates the robot's direction 90 degrees clockwise" $ do
+    describe "Rotating clockwise" $ do
```

Comment:
This is more of a comment on the canonical data than this particular PR - it may not be obvious to students that right is clockwise and left is counter-clockwise.